docs/concepts/tx-lifecycle.md

```diff
@@ -0,0 +1,196 @@
+
+### Byzantine Validators
+
+In a public blockchain network, it is possible for validators to be **byzantine**, or malicious, which may prevent `tx` from being committed in the blockchain. Some malicious behaviors are detectable, such as going offline (unavailable to vote) or voting on multiple blocks in one round - these behaviors are logged and can be penalized accordingly. Others are less obvious, such as the proposer deciding to censor the transaction by excluding it from the block.
+
+If `tx` is included in a block that the network commits to, it officially becomes part of the blockchain, which logs the application's transaction history.
+
+## State Changes
+
+During consensus, all full-nodes come to agreement on not only which transactions but also the precise order of the transactions in their block. However, apart from committing to this block in consensus, the ultimate goal is actually for full-nodes to commit to a new state generated by the transaction state changes.
+In order to execute `tx`, the following ABCI function calls are made in order to communicate to the application what state changes need to be made. While full-nodes each run everything individually, since the messages' state transitions are deterministic and the order was finalized during consensus, this process yields a single, unambiguous result.
+```
+		-----------------------		
+		| BeginBlock	      |        
+		-----------------------		
+		          |		
+			  v			
+		-----------------------		    
+		| DeliverTx(tx0)      |  
+		| DeliverTx(tx1)      |   	  
+		| DeliverTx(tx2)      |  
+		| DeliverTx(tx3)      |  
+		|	.	      |  
+		|	.	      |
+		|	.	      |
+		-----------------------		
+		          |			
+			  v			
+		-----------------------
+		| EndBlock	      |         
+		-----------------------
+		          |			
+			  v			
+		-----------------------
+		| Commit	      |         
+		-----------------------
+```
+#### BeginBlock
+
+[`BeginBlock`](./app-anatomy.md#beginblocker-and-endblocker) is run first, and mainly transmits important data such as block header and Byzantine Validators from the last round of consensus to be used during the next few steps. No transactions are handled here.
+
+#### DeliverTx
+
+The `DeliverTx` ABCI function defined in [`baseapp`](./baseapp.md) does the bulk of the state change work: it is run for each transaction in the block in sequential order as committed to during consensus. Under the hood, `DeliverTx` is almost identical to `CheckTx` but calls the [`runTx`](./baseapp.md#runtx) function in deliver mode instead of check mode. Instead of using their `checkTxState` or `queryState`, full-nodes select a new copy, `deliverTxState`, to deliver transactions:
```

Comment:
Recurring issue:

You use "for each transaction" in the paragraph, and then take the viewpoint of a single tx. You have to choose between the two.